common/views/themes/typography.ts

```diff
@@ -218,7 +218,7 @@ export const typography = css<GlobalStyleProps>`
       margin-bottom: 0;
     }
 
-    * + * {
+    > * + * {
```

Comment:
This keeps it to one level deep (to prevent the styles from affecting the injected hover card). I guess if we have e.g. a list inside a list and we wanted to inner items to have the same spacing as the outer items we'd have to handle that separately now (but I'm not sure we do have that situation anywhere)

Reply:
I think you're right. I don't _think_ we should care about that change being the result of this css change since it isn't a component that exists in prod? I can tidy up the story so that it doesn't look bad though.

Reply:
That's right, would be good to write it in a Prismic way so it would flag actual issues!